Cargo.toml

```diff
@@ -13,7 +13,7 @@ keywords = ["format", "parse", "encode"]
 
 [dependencies]
 anyhow = "1.0"
-clap = "~2.27.0"
+clap = { version = "4.0.17", features = ["cargo"] }
```

Comment:
🗺️ The `cargo` feature allows `clap` to pull information about the crate from its `Cargo.toml` file at compile time. This makes it possible to include the list of crate authors and the crate version in the `--help` message.